libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -63,6 +64,8 @@ @interface SFHybridViewController()
     SFHybridViewConfig *_hybridViewConfig;
 }
 
+@property (nonatomic, readwrite, assign) BOOL useWKWebView;
```

Comment:
@khawkins I thought about it briefly, but in `5.0` a lot of this will probably go away, so long term `UIWebView` subclass makes no sense anyway, at which time we'll have to combine them again. Hence, I opted for method level reuse as much as possible instead.